classes/locallib/config.php

```diff
@@ -266,6 +266,8 @@ public static function get_options() {
                'lockonjoinconfigurable_editable' => self::get('lockonjoinconfigurable_editable'),
                'lockonjoinconfigurable_default' => self::get('lockonjoinconfigurable_default'),
                'welcome_default' => self::get('welcome_default'),
+               'participant_guestlink' => self::get('participant_guestlink'),
```

Comment:
This should also have an entry in defaults

Reply:
As well as in get_all_settings (whatever its called)

Reply:
That's been added now.